bolides/bolide.py

```diff
@@ -17,50 +17,81 @@ class Bolide():
     def __init__(self, eventid):
         self.eventid = eventid
         self.json = self._load_json(eventid)['data'][0]
+        self.nSatellites = len(self.json['attachments'])
 
     def _load_json(self, eventid):
         """Returns a dictionary containing the data for the bolide."""
         url = f"{API_ENDPOINT_EVENT}/{eventid}"
         r = requests.get(url)
         return r.json()
 
+    @property
+    def detectedBy(self):
+        return self.json['detectedBy']
```

Comment:
Note: there is a widespread convention in the Python community to use names_with_underscores for variable and function names (e.g. `detected_by`), but given that the ATAP JSON data uses camelCase, it of course makes sense to make an exception to that convention here!